aten/src/ATen/native/quantized/cuda/affine_quantizer.cu

```diff
@@ -47,39 +62,37 @@ void dequantize_tensor_per_tensor_affine_cuda(
   AT_DISPATCH_QINT_TYPES(
       qtensor.scalar_type(), "dequantize_tensor_per_tensor_affine_cuda", [&]() {
         auto iter = TensorIteratorConfig()
-          .check_all_same_dtype(false)
-          .add_output(rtensor)
-          .add_input(qtensor)
-          .build();
+                        .check_all_same_dtype(false)
+                        .add_output(rtensor)
+                        .add_input(qtensor)
+                        .build();
         gpu_kernel(iter, [=] GPU_LAMBDA(scalar_t value) -> float {
           return (static_cast<float>(value.val_) - zero_point) * scale;
         });
       });
 }
 
 void quantize_tensor_per_channel_affine_cuda(
-  const Tensor& rtensor,
-  Tensor& qtensor,
-  const Tensor& scales,
-  const Tensor& zero_points,
-  int64_t axis) {
-
+    const Tensor& rtensor,
+    Tensor& qtensor,
+    const Tensor& scales,
+    const Tensor& zero_points,
+    int64_t axis) {
+  static const std::string fn_name = "quantize_tensor_per_channel_affine_cuda";
```

Comment:
what does this do?

Reply:
something weird happened to this PR, only like half the changes are here, this change was added because we use it both for the macro dispatch and the zero_point checks.

Reply:
ok, its fixed now